packages/common/src/hooks/chats/useAudiusLinkResolver.ts

```diff
@@ -26,16 +15,37 @@ const {
   instanceOfUserResponse
 } = ResolveApi
 
+// TODO: Move this to util file
+const escapeRegexSpecialChars = (str: string): string => {
+  const specialChars = /[.*+?^${}()|[\]\\]/g
+  return str.replace(specialChars, '\\$&')
+}
+
+export type LinkEntity =
+  | {
+      link: string
+      type: 'track'
```

Comment:
nit: this could be an enum; could maybe use `EntityType`?

Reply:
yea i was thinking of using `Kind` here, but seemed like overkill

Reply:
I think we decided to stop using enums as much and prefer plain strings? down to debate again, but that pattern seems to be the standard